### ISSUE TITLE
False positive "File annotations should be separated from file contents with a blank line"

### ISSUE THREAD

**Issue by ileasile** [pinterest/ktlint#914]
For a code 
```
@file:Suppress("UnstableApiUsage")

pluginManagement {

}
```

in `settings.gradle.kts` I receive "File annotations should be separated from file contents with a blank line" (experimental rules are turned on). Also, formatting adds two additional blank lines.

## Expected Behavior
There should be no exception

## Observed Behavior
Exception happens

## Steps to Reproduce
Example was given above

## Your Environment
* Version of ktlint used: Default for my Gradle plugin (seems to be 0.38.1)
* Name and version (or code for custom task) of integration used (Gradle plugin, Maven plugin, command line, custom Gradle task): Gradle plugin, org.jlleitschuh.gradle:ktlint-gradle:9.4.0
* Version of Gradle used (if applicable): 6.6.1
* Operating System and version: Windows 10
* Link to your project (if it's a public repository): -

**Comment by shashachu** [comment]
Introduced https://github.com/pinterest/ktlint/pull/714
